scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1372,4 +1372,9 @@
                                :reason_string "Preempted by rebalancer"
                                :sandbox_directory "/path/to/working/directory"
                                :status "success")]
-            (is (= expected-map (dissoc instance-map :start_time)))))))))
\ No newline at end of file
+            (is (= expected-map (dissoc instance-map :start_time)))))))))
+
+(deftest test-job-create-allowed?
```

Comment:
I care about the action and the entity. Imho, changing them should cause the test to fail.